### PR TITLE
(VELOCITY-874) Update Jenkins tutorial

### PR DESCRIPTION
**Pull request by sschneid**

### CODE REVIEW

File: 1.8/usage/tutorials/jenkins/index.md [hunk 1]
```diff
@@ -66,7 +66,8 @@ as NFS, CIFS, and so on.
 
 If you only want to run Jenkins in a development environment, it's trivial
 to pin it to a single agent in the DC/OS cluster. Create the file
-`options.json` with the configuration below:
+`options.json` with the configuration below, modifying `pinned-hostname` to
+correspond with an agent IP in your DC/OS cluster:
 
 ```bash
 $ cat options.json
```
Comment (ssk2) [inline 1.8/usage/tutorials/jenkins/index.md]: We should update this line to `storage.pinned-hostname` since the package has changed.

Comment (ssk2) [inline 1.8/usage/tutorials/jenkins/index.md]: Same goes for the production example below.

**Comment by ssk2** [comment]
Let's update the ToC to either remove `Creating a file share on Microsoft Azure` or  to include a corresponding line for EFS.

**Comment by ssk2** [comment]
A couple of small changes but otherwise looks good.

**Comment by sschneid** [comment]
Updated!

File: 1.7/usage/tutorials/jenkins/index.md [hunk 2]
```diff
@@ -66,13 +69,14 @@ as NFS, CIFS, and so on.
...
-`options.json` with the configuration below:
+`options.json` with the configuration below, modifying
+`storage.pinned-hostname` to correspond with an agent IP in your DC/OS cluster:
 
 ```bash
 $ cat options.json
 {
     "jenkins": {
-        "pinned-hostname": "10.100.100.88"
+        "storage.pinned-hostname": "10.100.100.88"
```
Comment (ssk2) [inline 1.7/usage/tutorials/jenkins/index.md]: Oh, sorry, I meant the top level property should be "storage" instead of "jenkins".

Comment (ssk2) [inline 1.7/usage/tutorials/jenkins/index.md]: See https://github.com/mesosphere/universe/blob/version-3.x/repo/packages/J/jenkins/11/config.json for reference.

Comment (sascala) [inline 1.7/usage/tutorials/jenkins/index.md]: LGTM. I made some little edits, mostly prepositions.

**Pull request merged**